.drone.star

```diff
@@ -1696,12 +1696,9 @@ def runWebuiAcceptanceTests(suite, alternateSuiteName, filterTags, extraEnvironm
 def redisService():
 	return[{
 		'name': 'redis',
-		'image': 'webhippie/redis',
+		'image': 'redis:6-alpine',
 		'pull': 'always',
-		'environment': {
-			'REDIS_DATABASES': 1
-		},
-	}]
+    }]
```

Comment:
Not sure how picky `.star` files are, but this line is still one or two spaces off 🥴

Reply:
They are not really picky. The problem is, that I don't want to run a formatter on that Starlark file because it causes a huge diff :-(. We should do that once in an extra PR or in CI to enforce a specific formatting.

Reply:
I see, would you mind opening a GitHub issue for this so we don't forget? :) could be a `good first issue` for outsiders, too

Reply:
Will open a PR now ;-)

Reply:
PR is in #4987